pkg/sensors/tracing/generictracepoint.go

```diff
@@ -1083,3 +1097,35 @@ func handleMsgGenericTracepoint(
 func (t *observerTracepointSensor) LoadProbe(args sensors.LoadProbeArgs) error {
 	return LoadGenericTracepointSensor(args.BPFDir, args.Load, args.Maps, args.Verbose)
 }
+
+// NormalizeSockaddrUnPath returns a display-friendly byte slice:
+// "@name" for abstract UNIX sockets (trimming trailing NUL padding),
+// or the pathname up to the first NUL for filesystem sockets.
+func NormalizeSockaddrUnPath(raw []byte) []byte {
```

Comment:
If we had a flag for abstract sockets then we could use that to spot an abstract socket.